sql/modules/Company.sql

```diff
@@ -14,7 +14,6 @@ DROP TYPE IF EXISTS company_entity CASCADE;
 
 CREATE TYPE company_entity AS(
```

Comment:
Since we are modifying the type 'company_entity' we should take the opportunity to add a SQL comment explaining the type and its use.

Reply:
Yep. Done.